docs/reference/mapping/types/boolean.asciidoc

```diff
@@ -180,7 +180,28 @@ The following parameters are accepted by `boolean` fields:
 
     Accepts any of the true or false values listed above. The value is
     substituted for any explicit `null` values. Defaults to `null`, which
-    means the field is treated as missing.
+    means the field is treated as missing. Note that this cannot be set
+    if the `script` parameter is used.
+
+`on_script_error`::
+
+    Defines what to do if the script defined by the `script` parameter
+    throws an error at indexing time. Accepts `reject` (default), which
+    will cause the entire document to be rejected, and `ignore`, which
+    will register the field in the document's
+    <<mapping-ignored-field,`_ignored`>> metadata field and continue
+    indexing. This parameter can only be set if the `script` field is
+    also set.
+
+`script`::
+
+    If this parameter is set, then the field will index values generated
+    by this script, rather than reading the values directly from the
+    source. If a value is set for this field on the input document, then
+    the document will be rejected with an error.
+    Scripts are in the same format as their
+    <<runtime-mapping-fields,runtime equivalent>>. Scripts can only be
+    configured on `long` and `double` field types.
```

Comment:
Don't need this last sentence for booleans :)

Reply:
ops